mdsuite/database/data_manager.py

```diff
@@ -102,15 +108,17 @@ def generator(batch_number: int, batch_size: int, database: str, data_path: list
                 if type(self.atom_selection) is dict:
                     select_slice = {}
                     for item in self.atom_selection:
-                        select_slice[item] = np.s_[self.atom_selection[item], start:stop]
+                        select_slice[item] = np.s_[self.atom_selection[item],
+                                             start:stop]
```

Comment:
I think this is particularly hard to read, because of the two `[]`.
Again I don't see a reason why two arguments should be split into two lines if it does not cut the line length.

Reply:
It cuts the pep8 length